Goal: fix some typos in the XML manpages.

Fixes: #352135, 369806

Status wrt upstream: Debian specific.

It is probably a bug of docbook-xsl 1.68.1-0.1

Index: shadow-4.0.18.1/man/passwd.1.xml
===================================================================
--- shadow-4.0.18.1.orig/man/passwd.1.xml	2006-08-25 11:56:01.738050007 +0200
+++ shadow-4.0.18.1/man/passwd.1.xml	2006-08-25 11:57:15.846652378 +0200
@@ -87,8 +87,8 @@
       <title>Hints for user passwords</title>
       <para>
 	The security of a password depends upon the strength of the
-	encryption algorithm and the size of the key space. The
-	<emphasis>UNIX</emphasis> System encryption method is based on the
+	encryption algorithm and the size of the key space.
+	The <emphasis>UNIX</emphasis> System encryption method is based on the
 	NBS DES algorithm and is very secure. The size of the key space
 	depends upon the randomness of the password which is selected.
       </para>
Index: shadow-4.0.18.1/man/useradd.8.xml
===================================================================
--- shadow-4.0.18.1.orig/man/useradd.8.xml	2006-08-25 11:56:53.106467569 +0200
+++ shadow-4.0.18.1/man/useradd.8.xml	2006-08-25 11:57:15.950653223 +0200
@@ -35,7 +35,7 @@
     <title>DESCRIPTION</title>
       <para>
 	When invoked without the <option>-D</option> option, the
-	<command>useradd</command> command creates a new user account using
+	 <command>useradd</command> command creates a new user account using
 	the values specified on the command line and the default values from
 	the system. Depending on command line options, the useradd command
 	will update system files and may also create the new user's home
